tests/sentry/web/test_api.py

```diff
@@ -159,7 +164,7 @@ def test_organization_url_region(self):
             assert data["isAuthenticated"] is True
             assert data["lastOrganization"] == self.organization.slug
             assert data["sentryUrl"] == "http://testserver"
-            assert data["organizationUrl"] == f"http://{self.organization.slug}.eu.testserver"
+            assert data["organizationUrl"] == f"http://eu.{self.organization.slug}.testserver"
```

Comment:
Do we need slug interpolation anymore? We could re-introduce it when we need it.